mlir/unittests/Analysis/utils.h

```diff
@@ -0,0 +1,41 @@
+//===- utils.h - utility functions for tests ------------------------------===//
```

Comment:
I think all file names in the MLIR codebase follow UpperCamelCase. It would be better to rename this file to "FACUtils.h" or "Utils.h".